app/helpers/books_helper.rb

```diff
@@ -0,0 +1,5 @@
+module BooksHelper
+  def latest(quantity = 3)
+    Book.order('created_at DESC').limit(quantity)
```

Comment:
@b-artem it's vey bad idea to use activerecord in helpers. If you need data in template - load it in controller

Reply:
Done